conbench/api/results.py

```diff
@@ -206,6 +217,12 @@ def post(self) -> None:
         return self.response_201_created(self.serializer.one.dump(benchmark_result))
 
 
+def make_json_response(data: bytes, status_code: int) -> f.Response:
+    # Note(JP): it's documented that a byte sequence can be passed in:
+    # https://flask.palletsprojects.com/en/2.2.x/api/#flask.Flask.make_response
+    return f.make_response((data, status_code, {"content-type": "application/json"}))
```

Comment:
Can use this helper elsewhere, and then also move it elsewhere.

I wanted to build a thin response generation wrapper which takes a byte sequence as input for the data in the response body.